build/RunTestsOnHelix.sh

```diff
@@ -8,3 +8,8 @@ export TestExecutionDirectory=$(pwd)/testExecutionDirectory
 mkdir $TestExecutionDirectory
 export DOTNET_CLI_HOME=$TestExecutionDirectory/.dotnet
 cp -a $HELIX_CORRELATION_PAYLOAD/t/TestExecutionDirectoryFiles/. $TestExecutionDirectory/
+
+# call dotnet new so the first run message doesn't interfere with the first test
+dotnet new
+# avoid potetial cocurrency issues when nuget is creating nuget.config
```

Comment:
small typo, should this be "concurrency"

Reply:
good catch, I just fixed it